add-on/chrome/test/mochitest/browser_LoopSidebar.js

```diff
@@ -0,0 +1,20 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/*
+ * This file contains tests for the LoopSidebar
+ */
+"use strict";
+
+const { LoopSidebar } = Cu.import("chrome://loop/content/modules/LoopSidebar.jsm", {});
+
+add_task(function* test_LoopSidebar_noSidebar() {
+  let sidebar = window.document.getElementById("loop-side-browser");
+  Assert.ok(!sidebar, "there should be no sidebars yet");
+});
+
+add_task(function* test_LoopSidebar_createSidebar() {
+  LoopSidebar.createSidebar(window);
```

Comment:
I'd suggest changing these tests to create the sidebar on a new window, and then close the window afterwards. That way we have less chance of causing side-effects on other tests.

There's functions in BrowserTestUtils.jsm that will help with this in e10s compatible ways.